src/material/BUILD.bazel

```diff
@@ -16,7 +16,7 @@ ts_library(
 
 filegroup(
     name = "overviews",
-    srcs = ["//src/material/%s:overview" % name for name in MATERIAL_ENTRYPOINTS],
+    srcs = ["//src/material/%s:overview" % name for name in MATERIAL_ENTRYPOINTS if not 'legacy' in name],
```

Comment:
This allows us to get rid of the overview files for the legacy components which won't be used.